Calculator/Calculator/Controller/CalculatorViewController.swift

```diff
@@ -186,4 +174,28 @@ extension CalculatorViewController {
             formulaListScrollView.setContentOffset(bottomOffset, animated: true)
         }
     }
+    
+    func addFormulaStackView(_ inputOperatorText: String) {
+        guard let numberLabelText = inputNumberLabel.text,
+              let operatorLabelText = inputOperatorLabel.text else { return }
+        
+        let formulaStackView = makeStackView()
```

Comment:
```suggestion
        let formulaStackView: UIStackView = {
              let formulaStackView: UIStackView = UIStackView()
        
              formulaStackView.axis = .horizontal
              formulaStackView.spacing = 8
        
              return formulaStackView
        }()
```
이렇게 사용할 수도 있을 것 같습니다.

Reply:
오오 밈! 감사합니다!
이렇게 바꾸면 인스턴스를 사용할때만 함수 초기화를 하겠네요!
수정하였습니다!

Reply:
해당 부분 추가 설명 부탁드립니다🥹